tfhe/src/integer/server_key/radix_parallel/tests_cases_comparisons.rs

```diff
@@ -1037,135 +1532,159 @@ macro_rules! define_scalar_comparison_test_functions {
     PARAM_MESSAGE_2_CARRY_2_KS_PBS,
     PARAM_MESSAGE_3_CARRY_3_KS_PBS,
     PARAM_MESSAGE_4_CARRY_4_KS_PBS
 });
 
-create_parametrized_test!(integer_smart_scalar_min_parallelized_256_bits {
+#[cfg(not(feature = "__coverage"))]
+create_parametrized_test!(integer_smart_scalar_min_parallelized_u256 {
     PARAM_MESSAGE_2_CARRY_2_KS_PBS,
-    // No test for 3_3, see define_comparison_test_functions macro
+    // No test for 3_3, see define_scalar_comparison_test_functions macro
     PARAM_MESSAGE_4_CARRY_4_KS_PBS
 });
 
-create_parametrized_test!(integer_smart_scalar_max_parallelized_256_bits {
+#[cfg(not(feature = "__coverage"))]
+create_parametrized_test!(integer_smart_scalar_max_parallelized_u256 {
     PARAM_MESSAGE_2_CARRY_2_KS_PBS,
-    // No test for 3_3, see define_comparison_test_functions macro
+    // No test for 3_3, see define_scalar_comparison_test_functions macro
     PARAM_MESSAGE_4_CARRY_4_KS_PBS
 });
 
-create_parametrized_test!(integer_scalar_min_parallelized_256_bits {
+#[cfg(not(feature = "__coverage"))]
+create_parametrized_test!(integer_scalar_min_parallelized_u256 {
     PARAM_MESSAGE_2_CARRY_2_KS_PBS,
-    // No test for 3_3, see define_comparison_test_functions macro
+    // No test for 3_3, see define_scalar_comparison_test_functions macro
     PARAM_MESSAGE_4_CARRY_4_KS_PBS
 });
 
-create_parametrized_test!(integer_scalar_max_parallelized_256_bits {
+#[cfg(not(feature = "__coverage"))]
+create_parametrized_test!(integer_scalar_max_parallelized_u256 {
     PARAM_MESSAGE_2_CARRY_2_KS_PBS,
-    // No test for 3_3, see define_comparison_test_functions macro
+    // No test for 3_3, see define_scalar_comparison_test_functions macro
     PARAM_MESSAGE_4_CARRY_4_KS_PBS
 });
 
+#[cfg(feature = "__coverage")]
+create_parametrized_test!(integer_unchecked_scalar_min_parallelized_u8 {
+    COVERAGE_PARAM_MESSAGE_2_CARRY_2_KS_PBS,
+});
+
+#[cfg(feature = "__coverage")]
+create_parametrized_test!(integer_unchecked_scalar_max_parallelized_u8 {
+    COVERAGE_PARAM_MESSAGE_2_CARRY_2_KS_PBS,
+});
+
+#[cfg(feature = "__coverage")]
+create_parametrized_test!(integer_smart_scalar_min_parallelized_u8 {
+    COVERAGE_PARAM_MESSAGE_2_CARRY_2_KS_PBS,
+});
+
+#[cfg(feature = "__coverage")]
+create_parametrized_test!(integer_smart_scalar_max_parallelized_u8 {
+    COVERAGE_PARAM_MESSAGE_2_CARRY_2_KS_PBS,
+});
+
+#[cfg(feature = "__coverage")]
+create_parametrized_test!(integer_scalar_min_parallelized_u8 {
+    COVERAGE_PARAM_MESSAGE_2_CARRY_2_KS_PBS,
+});
+
+#[cfg(feature = "__coverage")]
+create_parametrized_test!(integer_scalar_max_parallelized_u8 {
+    COVERAGE_PARAM_MESSAGE_2_CARRY_2_KS_PBS,
+});
+
 /// The goal of this function is to ensure that scalar comparisons
 /// work when the scalar type used is either bigger or smaller (in bit size)
 /// compared to the ciphertext
-fn integer_unchecked_scalar_comparisons_edge(param: ClassicPBSParameters) {
+fn test_unchecked_scalar_comparisons_edge(param: ClassicPBSParameters) {
```

Comment:
why was this renamed ?

with the create parameterized test below we will have a test called test_test_something ?

Reply:
You're right, it's a mistake.